<launch>
  <!-- Map server -->
  <arg name="map_file"/>
  <arg name="scan_topic" default="scan"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)"/>

  <!-- Localization -->
  <node name="RMI_pf" pkg="rmi_pf" type="pf_rmi_17.py" output="screen">
    <remap from="scan" to="$(arg scan_topic)"/>
  </node>
</launch>
